dls-controls/malcolmjs
# Feature/method viewer #270

## Description

Added a middle pane for methods, which shows a summary of past run statuses for the overall method and a plot/table similar to that used for an attribute for individual takes/returns parameters.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA/save
- view the middle pane
----- (below only works if running against a real Malcolm instance)
- type something into the 'design' field and click save
- view the updated info in the middle pane when the method returns
- click on the info icon next to the design field
- view the history table
- click on the plot tab at the bottom of the page
- view the plot

## Agile board tracking

connect to #270 
closes #270 
connect to #341 
closes #341 
connect to #301 
closes #301
connect to #336 
closes #336 
connect to #308 
closes #308
